Известно, что генеральная совокупность распределена нормально 
со средним квадратическим отклонением, равным 16. 
Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, 
если выборочная средняя M = 80, а объем выборки n = 256.

$\delta = \frac{t_{\lambda} \cdot \sigma}{\sqrt{n}}$

2Ф (t\_lambda) = lambda_

In [1]:
lambda_ = 0.95

Ф (t\_lambda) = 0.475

In [2]:
t_lambda = 1.96

In [6]:
delta = 1.96 * (16 / (256**0.5))

In [7]:
delta

1.96

In [8]:
80 - delta

78.04

In [9]:
80 + delta

81.96

Ответ (78.04 - 81.96)

In [3]:
import numpy as np

В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, 
получены опытные данные:
6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1
Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, 
оценить истинное значение величины X при помощи доверительного интервала, покрывающего это 
значение с доверительной вероятностью 0,95.

In [12]:
a = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

In [13]:
a_mean = a.mean()

In [15]:
a_std = (np.sum((a - a_mean)**2) / (10-1))**0.5
a_std

0.4508017549014448

In [16]:
t_kr = 2.262

In [17]:
delta = t_kr * a_std / (10**0.5)

In [18]:
delta

0.32246174408757394

In [19]:
a_mean - delta

6.267538255912426

In [20]:
a_mean + delta

6.912461744087575

ДИ = (6.2675, 6.9125)

Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. 
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр 
оказался равным 17.5 мм, а дисперсия известна и равна 4 мм.

H0: M = 17

H1: M > 17

U_nabl = delta \* (n\*\*0.5) / sigma

In [23]:
U_nabl = (17.5 - 17) * (100**0.5) / 4
U_nabl

1.25

In [24]:
alpha = 0.05

In [25]:
Ф = (1-2*alpha) / 2
Ф

0.45

In [26]:
u_kr = 1.645

In [27]:
U_nabl < u_kr

True

Принимаем H0

Продавец утверждает, что средний вес пачки печенья составляет 200 г. 
Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
Известно, что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%

In [4]:
a = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])

In [10]:
a_mean = a.mean()
a_mean

198.5

In [8]:
a_std = (np.sum((a - a_mean)**2) / (10-1))**0.5

In [9]:
a_std

4.453463071962462

H0 M = 200
H1 M < 200

In [16]:
u_kr = 3.250

In [13]:
U_nabl = (200 - a_mean) * (10**0.5) / (a_std**0.5)

In [14]:
U_nabl

2.2477206282842537

In [17]:
U_nabl < u_kr

True

Принимаем H0